docs/streams/developer-guide/processor-api.html

```diff
@@ -226,8 +226,9 @@ <h2>
                                 application instance, but must fit into the available local disk
                                 space.</li>
                             <li>RocksDB settings can be fine-tuned, see
-                                <a class="reference internal" href="config-streams.html#streams-developer-guide-rocksdb-config"><span class="std std-ref">RocksDB configuration</span></a>.</li>
-                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.PersistentKeyValueFactory.html">store variants</a>:
```

Comment:
This link is broken so I updated it to where I think it was intended to point, please check if it makes sense or not